x/merkledb/codec.go

```diff
@@ -41,13 +44,12 @@ var (
 	trueBytes  = []byte{trueByte}
 	falseBytes = []byte{falseByte}
 
-	errTooManyChildren    = errors.New("length of children list is larger than branching factor")
-	errChildIndexTooLarge = errors.New("invalid child index. Must be less than branching factor")
-	errLeadingZeroes      = errors.New("varint has leading zeroes")
-	errInvalidBool        = errors.New("decoded bool is neither true nor false")
-	errNonZeroKeyPadding  = errors.New("key partial byte should be padded with 0s")
-	errExtraSpace         = errors.New("trailing buffer space")
-	errIntOverflow        = errors.New("value overflows int")
+	ErrChildIndexTooLarge = errors.New("invalid child index. Must be less than branching factor")
+	ErrLeadingZeroes      = errors.New("varint has leading zeroes")
+	ErrInvalidBool        = errors.New("decoded bool is neither true nor false")
+	ErrNonZeroKeyPadding  = errors.New("key partial byte should be padded with 0s")
+	ErrExtraSpace         = errors.New("trailing buffer space")
+	ErrIntOverflow        = errors.New("value overflows int")
```

Comment:
Why are these exported now?

Reply:
If they can be returned externally, I felt like they should be visible externally

Reply:
Define external? The `encoderDecoder` interface and its methods are unexported. If a codec error gets bubbled up / returned to a caller outside this package, there's not really anything for them to do.